classes/cluster/mk20_stacklight_advanced/stacklight/server.yml

```diff
@@ -1,7 +1,7 @@
 classes:
 - system.collectd.client.remote
 - system.elasticsearch.server.cluster
-- system.influxdb.server.cluster
+- system.influxdb.server.single
```

Comment:
I'm currently reviewing all patches that were in master and not ported in dash. So this one is one of those and there is another one for installing InfluxDB on a single node. As we don't cherry-pick I will probably merge the two... it is coming, keep in touch.